src/olympia/lib/settings_base.py

```diff
@@ -1325,6 +1328,9 @@ def read_only_mode(env):
 MEDIA_ROOT = os.path.join(SHARED_STORAGE, 'uploads')
 TMP_PATH = os.path.join(SHARED_STORAGE, 'tmp')
 
+DATA_BACKUP_DIR = path('backups')
+DATA_BACKUP_INIT = '_init'
```

Comment:
imo the setting should be named something more like `DATA_BACKUP_INITIAL_DIR_NAME`, but I'm not sure it even needs to be a setting - seems more like it should be a constant.

Reply:
Fixed